nmp/all.nmp

```diff
@@ -2270,13 +2221,20 @@ op sbbu(src: LOC16)
 	}
 
 // DOC: page 401
-// NP: 4 ops
-// op 
-// 	syntax = "SBF 8bitOffset, EQ/NEQ/TC/NTC"
-// 	image = ""
-// 	action = {
+op sbf(instype: card(2), imm: card(8))
+	sbftype = 
+		if instype == 0b00 then "EQ"
+		else if instype == 0b01 then "NEQ"
+		else if instype == 0b10 then "TC" 
+		else "NTC" 
+		endif
+		endif
+		endif
```

Comment:
Use a switch here, GLISS2 will statically check that all cases are handled

```
sbftype = switch(insttype) {
 ...
```

Reply:
and so it is done